src/elementary/ReactionType/src/register.hpp

```diff
@@ -27,7 +26,7 @@ ReactionType::name_dictionary{
       { ReactionType::Entry{  38, "fission[3n]",    { "fission4", "n,3nf" } } },
       { ReactionType::Entry{ 102, "capture",        { "n,g", "n,gamma", "z,g", "z,gamma" } } },
       { ReactionType::Entry{  11, "2nd",            { "z,2nd", "n,2nd" }, { n, n, d } } },
-      { ReactionType::Entry{  16, "2n",             { "z,2n", "n,2n" }, { n, n } } },
+      { ReactionType::Entry{  16, "2n(t)",          { "z,2n", "n,2n" }, { n, n } } },
```

Comment:
Does `(t)` look too much like a triton?

Reply:
t as in total. it's a way to distinguish 16 from 875. When converting back from an identifier, it will never generate a 16 so the summation routines will always produce a 16